stdlib/2.7/sys.pyi

```diff
@@ -86,6 +86,9 @@ path_hooks = ...  # type: List[Any]
 path_importer_cache = ...  # type: Dict[str, Any]
 displayhook = ...  # type: Optional[Callable[[int], None]]
 excepthook = ...  # type: Optional[Callable[[type, BaseException, TracebackType], None]]
+exc_type = ...  # type: type
+exc_value = ...  # type: BaseException
```

Comment:
Strictly speaking, this can be an old-style class as well (and not derive from BaseException).

Declare as "Any"?